plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/tool/display/ToolProxy.java

```diff
@@ -199,8 +201,8 @@ else if (type.equals("actionTool")) //$NON-NLS-1$
         if (id.equals(DEFAULT_ID)) {
         	toolManager.defaultModalToolProxy = this;
         }
-        
-        
+
+        toggleButton = Boolean.parseBoolean(definition.getAttribute("toggeling"));
```

Comment:
getAttribute should use isToogleButton as dedined in exsd file

Reply:
`toggleButton = Boolean.parseBoolean(definition.getAttribute("isToggleButton"));`

would be correct since the attribute name is "isToggleButton" (see https://github.com/locationtech/udig-platform/pull/468/files#diff-70f7e778e59cb709746a2573c71406bd553f14bf3b4b9b6e5329e7bf3070a4d4)